### PR TITLE
[RW-1092] [risk = no] Review Right hand bar styling and functionality changes

### PR DESCRIPTION
**Pull request by smazumd**
Review Right hand bar styling and functionality changes

### CODE REVIEW

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.css [hunk 1]
```diff
@@ -29,3 +29,113 @@ input[type=text]:not([readonly]):focus {
...
+  padding-bottom: 0.7rem;
+}
+
+.items-date {
+  color:#224F83;
+  font-weight: bold;
+  padding-bottom: 0.15rem;
+}
+
+.items-Integer {
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.css]: Nit: Is there any reason the "I" is capitalized here? If not, we should change to lowercase to be consistent

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 2]
```diff
@@ -1,53 +1,53 @@
...
+               (ngModelChange)="myDate = $event" [name]="machineName" [id]="machineName" [formControl]="control"  type="date" clrDate>
+      </div>
     </div>
 
-    <div *ngSwitchCase="kinds.INTEGER">
-      <label [for]="machineName" class="flex-container">
-        <span class="flex-content-main">
+    <div *ngSwitchCase="kinds.INTEGER" class="text-container">
+      <label [for]="machineName" class="">
+        <span class="flex-content-main items-Integer">
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: Capital "I", same as above comment

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 3]
```diff
@@ -1,53 +1,53 @@
...
-  <ng-container [ngSwitch]="annotation.definition.annotationType">
+<div>
+  <div [ngSwitch]="annotation.definition.annotationType">
 
     <!-- STRING, DATE, INTEGER all rely on the blur event to update the annotation -->
     <div *ngSwitchCase="kinds.STRING">
-      <label [for]="machineName" class="flex-container">
-        <span class="flex-content-main">
+      <div class="text-container">
+        <label [for]="machineName" >
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: Remove whitespace

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 4]
```diff
@@ -1,53 +1,53 @@
...
           [id]="machineName"
           [formControl]="control">
         </textarea>
     </div>
 
     <div *ngSwitchCase="kinds.DATE">
-      <label [for]="machineName" class="flex-container">
-        <span class="flex-content-main">
+        <div class="text-container">
+            <label [for]="machineName" class="">
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: Remove empty class

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 5]
```diff
@@ -1,53 +1,53 @@
...
         </span>
-      </label>
-      <input
-        (blur)="handleInput()"
-        type="date"
-        [name]="machineName"
-        [id]="machineName"
-        [formControl]="control">
+                <span *ngIf="textSpinnerFlag" class="spinner spinner-inline spinner-md spin-in-overlay"> Loading...   </span>
+                <span><clr-icon *ngIf="successIcon" class="success is-solid" shape="success-standard" ></clr-icon></span>
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: Remove whitespace

**Comment by smazumd** [comment]
@lstrano-broad here goes all screen shots..

1. Review right panel view with all component

screen shot 2018-08-22 at 10 11 19 am

2. while updating data spinner and checked icons

screen shot 2018-08-22 at 10 11 52 am

screen shot 2018-08-22 at 10 12 01 am
3. Create Cohot wizard

screen shot 2018-08-22 at 10 13 18 am

4. Edit Cohort Wizard
screen shot 2018-08-22 at 10 14 48 am

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
+        <input class="modifier-calender" [ngModel]="myDate | date:'yyyy-MM-dd'"
+               (ngModelChange)="myDate = $event" [name]="machineName" [id]="machineName" [formControl]="control"  type="date" clrDate>
+      </div>
     </div>
 
-    <div *ngSwitchCase="kinds.INTEGER">
-      <label [for]="machineName" class="flex-container">
-        <span class="flex-content-main">
+    <div *ngSwitchCase="kinds.INTEGER" class="text-container">
+      <label [for]="machineName" class="">
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: remove empty class

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 7]
```diff
@@ -1,53 +1,53 @@
...
-        (blur)="handleInput()"
-        type="date"
-        [name]="machineName"
-        [id]="machineName"
-        [formControl]="control">
+                <span *ngIf="textSpinnerFlag" class="spinner spinner-inline spinner-md spin-in-overlay"> Loading...   </span>
+                <span><clr-icon *ngIf="successIcon" class="success is-solid" shape="success-standard" ></clr-icon></span>
+            </label>
+        </div>
+      <div (click)=" dateChange(myDate)" >
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: remove whitespace

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 8]
```diff
@@ -1,53 +1,53 @@
...
-      <label [for]="machineName" class="flex-container">
-        <span class="flex-content-main">
+    <div *ngSwitchCase="kinds.INTEGER" class="text-container">
+      <label [for]="machineName" class="">
+        <span class="flex-content-main items-Integer">
           {{ annotation.definition.columnName }}
-          <em>{{ datatypeDisplay | titlecase }}</em>
         </span>
+          <span *ngIf="textSpinnerFlag" class="spinner spinner-inline spinner-md spin-in-overlay"> Loading...   </span>
+          <span><clr-icon *ngIf="successIcon" class="success is-solid" shape="success-standard" ></clr-icon></span>
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: remove whitespace

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 9]
```diff
@@ -56,40 +56,47 @@
 
     <!-- ENUM and BOOLEAN rely on the change event to update the annotation -->
     <div *ngSwitchCase="kinds.ENUM">
-      <label [for]="machineName" class="flex-container">
-        <span class="flex-content-main">
+      <label [for]="machineName" class="">
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: remove empty class

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 10]
```diff
@@ -56,40 +56,47 @@
...
+        <span class="flex-content-main items-enum">
           {{ annotation.definition.columnName }}
-          <em>{{ datatypeDisplay | titlecase }}</em>
         </span>
+        <span *ngIf="textSpinnerFlag" class="spinner spinner-inline spinner-md spin-in-overlay"> Loading...   </span>
+          <span><clr-icon *ngIf="successIcon" class="success is-solid" shape="success-standard" ></clr-icon></span>
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: whitespace

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 11]
```diff
@@ -56,40 +56,47 @@
...
-          [formControl]="control">
-          <option value=''><em>No Selection</em></option>
-          <option *ngFor="let value of annotation.definition.enumValues" [value]="value">
-            {{ value | titlecase }}
-          </option>
-        </select>
+      <div>
+        <clr-dropdown  class="participant-dropdown">
+          <button type="button" class="btn btn-outline" clrDropdownTrigger>
+            <span *ngIf = "defaultAnnotation; else elseBlock">{{annotationOption}}</span>
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: whitespace

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 12]
```diff
@@ -56,40 +56,47 @@
...
           [id]="machineName"
           [formControl]="control">
-        <label [for]="machineName" class="flex-content-main">
-          {{ annotation.definition.columnName }}
-          <em>{{ datatypeDisplay | titlecase }}</em>
-        </label>
-      </div>
-    </div>
+        <span *ngIf="textSpinnerFlag" class="spinner spinner-inline spinner-md spin-in-overlay"> Loading...   </span>
+        <span><clr-icon *ngIf="successIcon" class="success is-solid" shape="success-standard" ></clr-icon></span>
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: whitespace

File: ui/src/app/cohort-review/participant-status/participant-status.component.html [hunk 13]
```diff
@@ -1,12 +1,24 @@
...
-  <div class="select">
-  <select [formControl]="statusControl">
-    <option *ngFor="let status of cohortStatusList" [value]="status.value">
-      {{status.display}}
-    </option>
-  </select>
+  <div>
+    <clr-dropdown  class="participant-dropdown">
+      <button type="button" class="btn btn-outline" clrDropdownTrigger>
+        <span *ngIf = "defaultOption; else elseBlock">{{participantOption}}</span>
```
Comment (dolbeew) [inline ui/src/app/cohort-review/participant-status/participant-status.component.html]: whitespace

File: ui/src/app/cohort-review/set-annotation-create/set-annotation-create.component.html [hunk 14]
```diff
@@ -1,79 +1,104 @@
...
-        <select id="kind" name="kind" formControlName="kind">
-          <option *ngFor="let dt of datatypes" [value]="dt.value">
-              {{ dt.displayName }}
-          </option>
-        </select>
-      </div>
+    <div class="annotation-dropdown">
+      <clr-dropdown >
+        <button type="button" class="btn btn-outline" clrDropdownTrigger>
+          <span *ngIf = "annotationOptions !== ''; else elseBlock">{{annotationOptions}}</span>
```
Comment (dolbeew) [inline ui/src/app/cohort-review/set-annotation-create/set-annotation-create.component.html]: whitespace

File: ui/src/app/cohort-review/set-annotation-create/set-annotation-create.component.html [hunk 15]
```diff
@@ -1,79 +1,104 @@
...
-        <button type="button"
-          class="btn btn-icon btn-sm"
-          (click)="addEnumValue()">
-          <clr-icon shape="plus" size="18"></clr-icon>
-        </button>
-      </div>
-      <ul>
-        <li *ngFor="let val of enumValues">
+      <label class="value-text" for="enum-values">Values</label>
+      <div class="">
```
Comment (dolbeew) [inline ui/src/app/cohort-review/set-annotation-create/set-annotation-create.component.html]: empty class

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 16]
```diff
@@ -1,53 +1,56 @@
...
-    <div *ngSwitchCase="kinds.INTEGER">
-      <label [for]="machineName" class="flex-container">
-        <span class="flex-content-main">
-          {{ annotation.definition.columnName }}
-          <em>{{ datatypeDisplay | titlecase }}</em>
-        </span>
-      </label>
+    <div *ngSwitchCase="kinds.INTEGER" class="text-container">
+        <div class="integer-container">
+            <label [for]="machineName" class="">
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: remove empty class

File: ui/src/app/cohort-review/annotation-item/annotation-item.component.html [hunk 17]
```diff
@@ -1,53 +1,56 @@
...
+            <span class="flex-content-main items-Integer">
+              {{ annotation.definition.columnName }}
+            </span>
+            <span *ngIf="textSpinnerFlag" class="spinner spinner-inline spinner-md spin-in-overlay"> Loading...   </span>
+            <span><clr-icon *ngIf="successIcon" class="success is-solid" shape="success-standard" ></clr-icon></span>
```
Comment (dolbeew) [inline ui/src/app/cohort-review/annotation-item/annotation-item.component.html]: whitespace

File: ui/src/app/cohort-review/set-annotation-modal/set-annotation-modal.component.html [hunk 18]
```diff
@@ -1,31 +1,20 @@
...
       <app-set-annotation-create (onFinish)="mode = 'list'"></app-set-annotation-create>
-    </div>
+  </div>
+</clr-modal>
 
-    <!-- Show the set annotation list -->
-    <div *ngIf="mode === 'list'">
+
+<!--This modal is for edit-->
+<clr-modal class="annotation-modal" [(clrModalOpen)]="openEdit" [clrModalClosable]="false" >
```
Comment (dolbeew) [inline ui/src/app/cohort-review/set-annotation-modal/set-annotation-modal.component.html]: whitespace

File: ui/src/app/cohort-review/set-annotation-modal/set-annotation-modal.component.html [hunk 19]
```diff
@@ -1,31 +1,20 @@
-<clr-modal
-  [(clrModalOpen)]="open"
-  [clrModalSize]="'lg'">
+<!--This modal is for create-->
+<clr-modal class="annotation-modal" [(clrModalOpen)]="open" [clrModalClosable]="false" >
```
Comment (dolbeew) [inline ui/src/app/cohort-review/set-annotation-modal/set-annotation-modal.component.html]: whitespace

**Pull request merged**